CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
I'm starting to think that the list of bug fixes really makes little sense and is not useful, as you said. So better remove it. Plus the way I was selecting bugs is not reliable, so it could create some confusion. If nobody objects, I'll remove the list from CHANGES, tag and release.